libs/template/cli/src/commands/create.ts

```diff
@@ -123,77 +134,58 @@ Please make sure to not use an existing directory name.`
     const archivePath = path.resolve(this.packageRoot, `template-${ref}.zip`);
 
     if (!fs.existsSync(archivePath)) {
-      const s = spinner();
-
-      s.start('Downloading latest template...');
-
       await this.nodeUtilService.downloadFile(
         this.getTempalteUrl(ref),
         archivePath
       );
-
-      s.stop('Template downloaded!');
     }
 
-    const s = spinner();
-
-    s.start('Extracting template...');
-
     await this.nodeUtilService.extractZip(archivePath, repoPath);
-
-    s.stop('Template extracted!');
   }
 
   protected async copyTemplate(
     options: CreateAppConfig,
     ref: OryxTemplateRef = OryxTemplateRef.Latest
   ): Promise<void> {
-    const s = spinner();
-
-    s.start('Copying template...');
-
     const repoPath = path.resolve(this.repoPath, ref, this.repoPaths[ref]);
 
     if (!fs.existsSync(repoPath)) {
       throw new Error(
-        'Template is not found! Please re-run the command again.'
+        'Application template is not found! Please re-run the command again.'
       );
     }
 
-    const templatePath = path.resolve(
-      repoPath,
-      'apps',
-      this.getTemplateFolder(options.preset)
-    );
+    await this.nodeUtilService.copyFolder(repoPath, options.path);
+  }
 
-    await this.nodeUtilService.copyFolder(templatePath, options.path);
+  protected async npmInstall(path: string): Promise<void> {
+    const s = spinner();
 
-    s.stop('Template copied!');
+    s.start('Installing dependencies...');
+    await this.nodeUtilService.executeCommand('npm install', path);
+    s.stop('Dependencies installed');
   }
 
   protected getTempalteUrl(ref: OryxTemplateRef): string {
     return this.repoUrl.replace('{ref}', this.repoRefs[ref]);
   }
 
-  protected getTemplateFolder(preset: OryxPreset): string {
-    switch (preset) {
-      case OryxPreset.B2C:
-        return 'storefront';
-      case OryxPreset.Fulfillment:
-        return 'fulfillment';
-    }
-  }
+  // WIP
+  // protected getTemplateFolder(preset: OryxPreset): string {
+  //   switch (preset) {
+  //     case OryxPreset.B2C:
+  //       return 'storefront';
+  //     case OryxPreset.Fulfillment:
+  //       return 'fulfillment';
+  //   }
+  // }
 
   protected promptName(): Promise<string> {
     return this.promptValue(
       text({
-        message: `What is the name of your app? ${c.dim(`[--name, -n]`)}`,
-        validate: (value) => {
-          if (!value.trim()) {
-            return 'Please enter a name';
-          }
-          return undefined;
-        },
+        message: `What is the name of your app?`,
+        placeholder: 'oryx-app',
+        defaultValue: 'oryx-app',
```

Comment:
Maybe `initialValue` will be better?

Reply:
initial you'll have to edit if you want to use different name, while default will be used if no name is provided, or you can just write your own